core/templates/pages/exploration-editor-page/services/router.service.spec.ts

```diff
@@ -147,7 +147,9 @@ describe('Router Service', () => {
 
     // @ts-ignore
     var jQuerySpy = spyOn(window, '$');
+    // @ts-ignore
```

Comment:
Why this was not an issue before?

Reply:
I'm not sure, but the error is the same as on the previous line.